app/routes/forms/new.js

```diff
@@ -3,22 +3,31 @@ import { inject as service } from '@ember/service';
 import { CONCEPT_STATUS } from '../../models/submission-document-status';
 
 export default class FormsNewRoute extends Route {
-  @service currentSession
+  @service currentSession;
+  @service store;
+  @service router;
 
   async beforeModel() {
-    const conceptStatuses = await this.store.query('submission-document-status', {
-      page: { size: 1 },
-      'filter[:uri:]': CONCEPT_STATUS
-    });
+    const conceptStatuses = await this.store.query(
+      'submission-document-status',
+      {
+        page: { size: 1 },
+        'filter[:uri:]': CONCEPT_STATUS,
+      }
+    );
 
     if (conceptStatuses.length)
       this.conceptStatus = conceptStatuses.firstObject;
   }
 
   async model() {
+    await this.currentSession.ensureLoaded();
```

Comment:
@benjay10 @Windvis I am not sure this is needed. We don't do this in loket. I would expect it to be fixed on level of the application route

Reply:
Yea, once the session service hook is implemented this won't be needed anymore.

Reply:
ok! sorry  missed part of the review
@benjay10 this is also the reason why such flaky behavior when creating new submission. The bestuurseenheid is sometimes empty, hence everything else crashes in enrich submissions

Reply:
@cecemel Yes that is exactly what I wrote about in the commit message and the above comment.

Reply:
@Windvis Thanks, this works!